doc/source/releases/23.2.rst

```diff
@@ -0,0 +1,950 @@
+
+.. to_doc
+
+23.2
+===============================
+
+.. announce_start
+
+Enhancements
+-------------------------------
+
+.. major_feature
+
+
+.. feature
+
+* Make form repeat blocks reordarable
```

Comment:
Those are PRs that already went into 23.1 ...

Reply:
Which ones?  They're all milestoned 23.2, and checking at least the first few (with `git branch -a --contains <blah>`) I don't see any issues.